addons/full-import/configuration.functions

```diff
@@ -0,0 +1,159 @@
+#!/bin/bash
+
+function config_files() {
+  files=`perl -I/usr/local/pf/lib_perl5/lib -I/usr/local/pf/lib -Mpf::file_paths -e 'print join("\n", @pf::file_paths::stored_config_files) . "\n"'`
+  files="$files /usr/local/pf/conf/pfconfig.conf"
+  files="$files `cat add_files.txt`"
+  echo $files
+}
+
+function restore_config_files() {
+  echo "Restoring configuration files"
+  dump_dir="$1"
+  files=`config_files`
+  for f in $files; do
+    echo "Restoring $f"
+    mkdir -p `dirname $f`
+    check_code $?
+    if [ -f $dump_dir$f ]; then
+      cp -a "$dump_dir$f" $f
+      check_code $?
+      chown pf: $f
+      check_code $?
+    else
+      echo "WARN: Unable to find $f in the restore directory"
+    fi
+  done
+}
+
+function list_pf_interfaces() {
+  perl -MConfig::IniFiles -I/usr/local/pf/lib_perl/lib/perl5/ -e "print join(\"\n\", Config::IniFiles->new( -file => '/usr/local/pf/conf/pf.conf')->GroupMembers('interface')) . \"\n\"" | sed 's/^interface //g'
+}
+
+function get_os_ip_address() {
+  ip a show dev $1 | egrep -o 'inet [0-9]+\.[0-9]+\.[0-9]+\.[0-9]+' | head -1 | sed 's/^inet //g'
```

Comment:
```
# ip -br -o a show dev eth0
eth0             UP             10.0.2.15/24 fe80::a00:27ff:fecb:8dc8/64
```
It could be simplify to avoid usage of regex.